docs/src/Box.doc.js

```diff
@@ -419,11 +419,54 @@ card(
       />
     </MainSection.Subsection>
     <MainSection.Subsection
-      title="Using role"
+      title="Using 'as' property"
       description={`
-    Setting the  \`role\` property on Box classifies the Box as the semantically appropriate HTML element. For example, setting \`role="banner"\` will designate that Box to be the equivalent of a \`<header>\` within the page, allowing assistive technology to classify the Box appropriately.
+      By default, the Box component renders a \`div\` element, which doesn't provide much meaning to the user or assistive technology. Use the \`as\` prop to inform which HTML component should be rendered by the Box component to ensure a meaningful experience.
 
-    Using the \`role\` property can prevent an excess of generic \`<div>s\` and give the user better context. Learn more about [ARIA roles](https://www.w3.org/TR/wai-aria/#usage_intro).
+      When using a Box component as a custom element, it is your responsibility to address all the accessibility implications. Both the \`role\` and \`as\` properties semantically classify the Box; however, the \`as\` prop defines a more concise way to describe the HTML element by modifying the underlying DOM element directly, which helps support both accessibility and SEO. We recommend using the \`as\` prop when possible, making sure that the prop type is semantically associated with the Box content.
+
+      Review the [available options for the as prop](#Props). For some of the options, like \`nav\`, you will also need to specify a \`title\` to ensure unique landmarks on the page.
```

Comment:
Learn more about [semantics in HTML](https://developer.mozilla.org/en-US/docs/Glossary/Semantics#semantics_in_html).

Reply:
I like adding a link here. Nice call!